samples/angular-todo/src/app/app.component.ts

```diff
@@ -69,7 +69,7 @@ export class AppComponent implements OnInit {
 
     Config.set({
       clientId: environment.WEB_OAUTH_CLIENT,
-      redirectUri: environment.APP_URL,
+      redirectUri: `${window.location.origin}/callback`,
```

Comment:
Let's make sure we're consistent with our use of this. In other location, you use `${window.location.href}callback`.